docs/components/custom/MarkdownHeading/__tests__/MarkdownHeading.spec.jsx

```diff
@@ -6,7 +6,9 @@ import MarkdownHeading from '../MarkdownHeading'
 
 describe('MarkdownHeading', () => {
   it('renders', () => {
-    const markdownHeading = shallow(<MarkdownHeading level="h1">Random Heading</MarkdownHeading>)
+    const markdownHeading = shallow(
+      <MarkdownHeading level="h1">{['Random Heading']}</MarkdownHeading>
+    )
```

Comment:
Please update this test to not pass an array as the child. Using a string is fine. Passing an array as the child doesn't really make sense